src/Serilog.Sinks.Udp/Sinks/Udp/Private/UdpClientWrapper.cs

```diff
@@ -23,25 +23,26 @@ internal class UdpClientWrapper : IUdpClient
     {
         private readonly UdpClient client;
 
-        public UdpClientWrapper(
-            int localPort,
-            IPAddress remoteAddress)
+        public UdpClientWrapper(int localPort)
         {
             if (localPort < IPEndPoint.MinPort || localPort > IPEndPoint.MaxPort)
                 throw new ArgumentOutOfRangeException(nameof(localPort));
-            if (remoteAddress == null)
-                throw new ArgumentNullException(nameof(remoteAddress));
 
             client = localPort == 0
-                ? new UdpClient(remoteAddress.AddressFamily)
-                : new UdpClient(localPort, remoteAddress.AddressFamily);
+                ? new UdpClient(AddressFamily.InterNetworkV6)
```

Comment:
This cannot be correct, we can't migrate over to IPv6. If you need IPv6 instead of IPv4 I think we need to solve that in another PR.

Reply:
While it's not overtly important since we don't do any of the listening part, you're correct. I'll modify it to use IPv4 mapped via IPv6 (which is the recommended stack on most networks)

Reply:
I'll test the change on my end as well, but I don't want it to be a part of this PR, it has nothing to do with the issue you've solved.

We'll do the change on a new feature branch. I'll revert this code for now.